sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap-from-image.js

```diff
@@ -138,7 +117,7 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
         // Check a few pixels near the top and bottom and make sure they have
         // the right color.
-        var tolerance = 10;
+        var tolerance = 8;
```

Comment:
I suspect the reason the tolerance needs to be so high is that the ImageBitmap constructor argument colorspaceConversion='none' is not supported yet. Other than that, the tolerances should be very close to 0. Please see sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image.js. Please give yourself a TODO to look at this again once that constructor argument is supported, and further, to use it in these tests.